data-serving/data-service/test/controllers/case.test.ts

```diff
@@ -105,11 +105,6 @@ describe('GET', () => {
             // Simulate index creation used in unit tests, in production they are
             // setup by the setup-db script and such indexes are not present by
             // default in the in memory mongo spawned by unit tests.
-            await mongoose.connect(process.env.MONGO_URL || '', {
```

Comment:
Should the comment above this be removed too?

Reply:
no this is still needed, the new connections was not needed.